assets/css/main.css

```diff
@@ -403,11 +403,72 @@ html, body {
 .module-panel table {
     margin: 0;
 }
-.news-panel .list-group-item-text {
-  padding: 10px;
+
+/* course news */
+.news-panel {
+    margin-bottom: 30px;
+}
+
+.future-instance {
```

Comment:
Should this be `.news-panel .future-instance {` to ensure it doesn't affect wrong places (future instance is quite generic name). In addition, it would make it more clear, where this class typically exists.

Reply:
I added these for clarification